### PR TITLE
ci: use just one smoketest docker compose file for optimize

### PR DESCRIPTION
**Pull request by misiekhardcore**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

**Review by grlimacan — APPROVED** [review]
Looks good to me, but please make sure to fulfill the pre-requisites of the two check boxes before merging

**Comment by misiekhardcore** [comment]
@grlimacan we are makin changes here to our own CI, which isn't influencing any other team, so we don't need to get any approvals from CI DRI

**Pull request merged**
